doc/overview.md

```diff
@@ -153,9 +153,11 @@ Development tools using the GCC C++ compiler:
 Build dependencies:
 
 ```
-# xbps-install fontconfig-devel libjpeg-turbo-devel libXext-devel libpng-devel libXft-devel libXpm-devel libXrandr-devel libXinerama-devel
+# xbps-install fontconfig-devel libjpeg-turbo-devel libXext-devel libpng-devel libXft-devel libXpm-devel libXrandr-devel libXinerama-devel pango-devel
 ```
 
+However its preferable to use xbps-src to build from template
```

Comment:
Would you mind adding an example on how to do this while at it? (If it isn't too lengthy that is)

Reply:
Guess I could add url for the 
quick-start guide?
https://github.com/void-linux/void-packages#quick-start

Reply:
to build from template, see the [quick-start](https://github.com/void-linux/void-packages#quick-start) for details probably would do the trick yeah.